tox.ini

```diff
@@ -40,4 +40,5 @@ commands =
 deps =
   test
 commands=
+  poetry install --quiet --no-root --no-interaction
```

Comment:
Why is this line newly needed? Shouldn't `tox` handle installing the dependencies out of the box?

Reply:
This is to handle the case where people add new requirements and other devs don't recreate their tox envs.

This is a no-op if the requirements are up-to-date